extensions/panache/hibernate-orm-panache-kotlin/runtime/src/main/kotlin/io/quarkus/hibernate/orm/panache/kotlin/PanacheQuery.kt

```diff
@@ -148,6 +152,53 @@ interface PanacheQuery<Entity: Any> {
      */
     fun withHint(hintName: String, value: Any): PanacheQuery<Entity>
 
+    /**
+     * <p>
+     * Enables a Hibernate filter during fetching of results for this query. Your filter must be declared
+     * with [FilterDef] on your entity or package, and enabled with [Filter] on your entity.
+     * <p>
+     * WARNING: setting filters can only be done on the underlying Hibernate [Session] and so this
+     * will modify the session's filters for the duration of obtaining the results (not while building
+     * the query). Enabled filters will be removed from the session afterwards, but no effort is made to
+     * preserve filters enabled on the session outside of this API.
+     * 
+     * @param filterName The name of the filter to enable
+     * @param parameters The set of parameters for the filter, if the filter requires parameters
+     * @return this query
```

Comment:
We usually describe the return type as `this query, modified`

Reply:
Ah, OK.

Reply:
Done.